textworld/generator/inform7/world2inform7.py

```diff
@@ -768,7 +761,7 @@ def gen_source(self, seed: int = 1234) -> str:
 
         Turning on the extra description command option is an action applying to nothing.
         Carry out turning on the extra description command option:
-            Decrease turn count by 1;
+            Decrease turn count by 1;  [This should not affect turn count.]
```

Comment:
I get this, but on its own this line does look kinda funny ;)

Reply:
Haha, I see that now :P. Do you have a suggestion?

Reply:
Maybe "internal framework commands shouldn't count as a turn"?